pallets/gear/src/internal.rs

```diff
@@ -330,6 +330,30 @@ where
         }
     }
 
+    pub(crate) fn charge_gas_for_dispatch_stash_hold(
+        id: impl Into<GasNodeIdOf<GasHandlerOf<T>>>,
+        stored_bn: u64,
+    ) {
+        let current_bn: u64 = Self::block_number().unique_saturated_into();
+
+        let delayed_block_amount = current_bn.saturating_sub(stored_bn);
+
+        // cost_of_block * (delay + reserve_for)
+        let gas_amount = delayed_block_amount
+            .saturating_add(CostsPerBlockOf::<T>::reserve_for().unique_saturated_into())
+            .saturating_mul(CostsPerBlockOf::<T>::dispatch_stash());
+
+        // Spending gas, if need.
+        if !gas_amount.is_zero() {
+            Self::spend_gas(id, gas_amount)
+        }
+    }
+
+    pub(crate) fn unlock_gas_tree_node(msg: MessageId) {
```

Comment:
Eliminate function

Reply:
It's better for readaility, 'cause this function has its defined purpose

Reply:
Function literally consists of 2 lines. You just hide the fact it might panic

Reply:
done